lib/server-code/runners/cloud-workers-broker.js

```diff
@@ -15,6 +16,31 @@ const Events = {
   TASK_PROCESSED     : 'TASK_PROCESSED',
 }
 
+function getKilledWorkerDetails(worker) {
+  const { appId, task } = worker
+  const details = [`appId: ${appId}`]
+
+  if (task.deploymentModelName) {
+    details.push(`deployed model: ${task.deploymentModelName}`)
+  }
+
+  if (task.className) {
+    const taskClassName = task.className.match(/\.\w+/)[0].substring(1)
```

Comment:
it seems like only services have this property, so maybe we just can replace this part:

```
const service = task.className.replace('services.', '')
```

does it make sense?

Reply:
if 'services' will change on somthing else like 'service' 'serviceName'

Reply:
I personally think it's reasonable

Reply:
we will fix it =)

but, at least we will still see the entire className as `service.FooBar`